lib/scrabble.rb

```diff
@@ -9,5 +11,65 @@ def point_values
       "U"=>1, "V"=>4, "W"=>4, "X"=>8,
       "Y"=>4, "Z"=>10
     }
+    @caps = []
+  end
+
+  def score(word)
+    if word.class == String
+      caps_letters = word.upcase.chars
+      caps_letters.each do |letter|
+        @caps << @point_values[letter]
```

Comment:
No need to use an instance variable here. Would need to create a `caps` variable outside of the each block, but aim to use a local variable where possible.